### PR TITLE
Swedish locale

### PR DESCRIPTION
**Pull request by JesperWe**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by benjycui** [comment]
It will be better if you can provide a complete locale file, like this: https://github.com/ant-design/ant-design/blob/master/components/locale-provider/en_US.tsx

**Comment by JesperWe** [comment]
This is my plan, but I can't do it until the rc-calendar devs have accepted https://github.com/react-component/calendar/pull/194

**Comment by benjycui** [comment]
Thanks, and you could remove `[WIP]` after you finish your plan 😄

File: components/date-picker/locale/sv_SE.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const locale = {
+  lang: assign({
+    placeholder: 'Välj datum',
+    rangePlaceholder: ['Startdatum', 'Slutdatum'],
+  }, CalendarLocale),
+  timePickerLocale: assign({}, TimePickerLocale),
+};
+
+// All settings at:
+// https://github.com/ant-design/ant-design/issues/424
```
Comment (afc163) [inline components/date-picker/locale/sv_SE.tsx]: @benjycui This comment may need to be updated.

**Comment by JesperWe** [comment]
Should be complete now. Depends on new version of rc-calendar and rc-pagin.ation

**Comment by benjycui** [comment]
Sorry, I make a mistake, you should PR to branch `feature-2.7`, thanks.

**Pull request closed without merging**